modules/ROOT/assets/images/cbm-architecture.svg

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="2649px" height="1338px" viewBox="0 0 2649 1338" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" style="background: #FFFFFF;">
+    <!-- Generator: Sketch 58 (84663) - https://sketch.com -->
+    <title>cbm-architecture</title>
+    <desc>Created with Sketch.</desc>
+    <defs>
+        <rect x="146" y="114" width="66" height="125" id="rect-1"></rect>
+        <rect x="146" y="114" width="66" height="125" id="rect-2"></rect>
+        <rect x="131" y="119" width="97" height="101" id="rect-3"></rect>
+        <rect x="131" y="119" width="97" height="101" id="rect-4"></rect>
+        <rect x="131" y="119" width="97" height="101" id="rect-5"></rect>
+    </defs>
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="cbm-architecture">
+            <g id="background">
+                <rect id="Rectangle" x="0" y="0" width="2649" height="1338"></rect>
+            </g>
+            <g id="intranet" transform="translate(756.000000, 96.000000)">
+                <path d="M10,0 L1783,0 C1788.52285,-1.01453063e-15 1793,4.4771525 1793,10 L1793,1143 C1793,1148.52285 1788.52285,1153 1783,1153 L10,1153 C4.4771525,1153 6.76353751e-16,1148.52285 0,1143 L0,10 C-6.76353751e-16,4.4771525 4.4771525,1.01453063e-15 10,0 Z" id="Rectangle" stroke="#000000" stroke-width="4" stroke-dasharray="15"></path>
+                <g id="sdks" transform="translate(511.000000, 45.000000)">
+                    <rect id="Rectangle" stroke="#000000" stroke-width="3" fill="#FFFFFF" x="0" y="0" width="359" height="311" rx="10"></rect>
+                    <text id="Web-Server" font-family="Helvetica" font-size="30" font-weight="normal" letter-spacing="-0.18461538" fill="#000000">
+                        <tspan x="100.997296" y="48">Web Server</tspan>
+                    </text>
+                    <text id="80" font-family="Helvetica" font-size="30" font-weight="normal" letter-spacing="-0.18461538" fill="#000000">
+                        <tspan x="162.500045" y="83">80</tspan>
```

Comment:
Will all web clients communicate with their web server using port 80? Do we need to add "(typical)"?

Reply:
- I've added the "port" suffix to make clear we're talking about the port numbers.
- Added port 443 for https

ports 80/443 are the defaults for http/https respectively, similarly to how 8091 is the default for CBServer and 4984 for Sync Gateway. I'm going to create a new PR to say the diagram is for information only and different architectures could be designed.

https://github.com/couchbase/docs-sync-gateway/commit/9679302d8ae248fc438940342e02910a312cb907?short_path=6e22201#diff-6e222012ced9fbb0cafa593feb59b5c8